tests/test_indication.py

```diff
@@ -0,0 +1,92 @@
+from manim import *
+
+
+class FocusOnTest(Scene):
+    def construct(self):
+        square = Square()
+        self.add(square)
+
+        self.play(FocusOn(square))
+
+
+class IndicateTest(Scene):
+    def construct(self):
+        square = Square()
+
+        self.play(Indicate(square))
+
+
+class FlashTest(Scene):
+    def construct(self):
+        square = Square()
+        self.add(square)
+
+        self.play(Flash(square))
+
+
+class CircleIndicateTest(Scene):
+    def construct(self):
+        square = Square()
+        self.add(square)
+
+        self.play(CircleIndicate(square))
+
+
+class ShowPassingFlashTest(Scene):
+    def construct(self):
+        square = Square()
+        self.add(square)
+
+        self.play(ShowPassingFlash(square))
+
+
+class ShowCreationThenDestructionTest(Scene):
+    def construct(self):
+        square = Square()
+
+        self.play(ShowCreationThenDestruction(square))
+
+
+class ShowCreationThenFadeOutTest(Scene):
+    def construct(self):
+        square = Square()
+
+        self.play(ShowCreationThenFadeOut(square))
+
+
+class ShowPassingFlashAroundTest(Scene):
+    def construct(self):
+        circle = Circle()
+
+        self.play(ShowPassingFlashAround(circle))
+
+
+class ApplyWaveTest(Scene):
+    def construct(self):
+        square = Square()
+
+        self.play(ApplyWave(square))
+
+
+class WiggleOutThenInTest(Scene):
+    def construct(self):
+        square = Square()
+        self.add(square)
+
+        self.play(WiggleOutThenIn(square))
+
+
+class TurnInsideOutTest(Scene):
+    def construct(self):
+        square = Square()
+        self.add(square)
+
+        self.play(TurnInsideOut(square))
+
+def test_scenes(get_config_test, Tester):
+    CONFIG = get_config_test
+    module_name = os.path.splitext(os.path.basename(__file__))[0].replace('test_', '')
+    for _, scene_tested in inspect.getmembers(sys.modules[__name__], lambda m: inspect.isclass(m) and m.__module__ == __name__):
+        Tester(scene_tested, CONFIG, module_name).test()
```

Comment:
This code is repeated among many test files. It should be refactored into a single function perhaps inside `tests/utils.py` and imported and executed inside each `test_scenes` at the end of each file.

Reply:
That's why I wanted to do but ... I'm not sure how to do that and not sure if this is even possible. 
Because of pytest, I can't call a function at the end of each module, at least because of the fixtures, that won't work (they will be marked as undefined). 
If you have an idea of a pythonic good way to do it I'm totally in favour.

Reply:
tests/utils.py
```
def utils_test_scenes(get_config_test, Tester):
    CONFIG = get_config_test
    module_name = os.path.splitext(os.path.basename(__file__))[0].replace('test_', '')
    for _, scene_tested in inspect.getmembers(sys.modules[__name__], lambda m: inspect.isclass(m) and m.__module__ == __name__):
        Tester(scene_tested, CONFIG, module_name).test()
```

Then, in all test files:
```
from utils import utils_test_scenes
def test_scenes(get_config_test, Tester):
    utils_test_scenes(get_config_test, Tester)
```

Would this work?

Reply:
No, the parameters `module_name `and the list of the scenes to tests are not passed to `utils_test_scenes`
We could just pass as parameter the current module to have access to its __file__ and stuff that I used (I don't know if this is possible and not certain whether this is a good practice)
or, we put pass to `utils_test_scenes` `module_name `, `list fo the scenes to test`, `get_config_test`, and `Tester` as parameters which .. will be almost as long as what was before. 
But maybe this is more pythonic.
What do you think ?

Reply:
How about something like the following:

```
from utils import utils_test_scenes, get_module_name, get_list_of_scenes_to_test

def test_scenes(get_config_test, Tester):
    utils_test_scenes(get_config_test, Tester, get_module_name(__file__), get_list_of_scenes_to_test(__name__))
```
Or something along those lines? In this case, `test_scenes` is still very dumb and far less prone to bugs even if it's repeated in many files (ie it's basically biolerplate), and all the actual code is in a single place in the utils file. (I'm not sure if this exact code can be made functional but I hope you get the gist.)

Reply:
I did that, and you were right, this is much better.

Reply:
Basically I moved `SceneTester` to `testing_utils.py`. 
I also created `get_scenes_to_test`, but left the module name to be enter manually in each module (it's simpler and just as effective). 
But I didn't touch the config stuff that much, since, as what I understood you will refactor it when this PR is merged.

Reply:
LGTM